docker/ol-solr-updater-start.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Can you confirm this file has the right execution permissions? :P

Reply:
% `ls -l docker/*.sh`
```
-rwxr-xr-x  1 cclauss  staff  106 Dec 14 17:57 docker/docker-entrypoint.sh
-rwxr-xr-x  1 cclauss  staff  128 Dec 14 17:57 docker/ol-covers-start.sh
-rwxr-xr-x  1 cclauss  staff  359 Dec 14 17:57 docker/ol-db-init.sh
-rwxr-xr-x  1 cclauss  staff  692 Dec 16 17:05 docker/ol-home-start.sh
-rwxr-xr-x  1 cclauss  staff   93 Dec 14 17:57 docker/ol-infobase-start.sh
-rwxr-xr-x  1 cclauss  staff  150 Dec 16 17:05 docker/ol-solr-updater-start.sh
-rwxr-xr-x  1 cclauss  staff  244 Dec 14 17:57 docker/ol-web-start.sh
```